engine/src/main/scala/cromwell/engine/workflow/WorkflowActor.scala

```diff
@@ -125,31 +126,83 @@ object WorkflowActor {
 
   case object Restart extends WorkflowActorMessage with StartMode {
```

Comment:
Agree w/ @mcovarr provided that we actually intend to go back after the structure is settled a bit more and make messages actually be messages